packages/utils/src/lib/id.ts

```diff
@@ -0,0 +1,100 @@
+/**
```

Comment:
nit: for licenses do `/*!` to preserve the license comment from munging (we use this in other places in the codebase too)